README.md

```diff
@@ -18,12 +18,19 @@ Add the following to your `build.sbt`:
 resolvers += Resolver.jcenterRepo
 
 // akka 2.5.x
-libraryDependencies += "com.github.dnvriend" %% "akka-persistence-jdbc" % "3.2.0"
+libraryDependencies += "com.github.dnvriend" %% "akka-persistence-jdbc" % "3.3.0"
 
 // akka 2.4.x
 libraryDependencies += "com.github.dnvriend" %% "akka-persistence-jdbc" % "2.8.0"
 ```
 
+## Release notes
+
+Latest release notes can be found [here](https://github.com/dnvriend/akka-persistence-jdbc/releases/latest).
```

Comment:
You can change the link to https://github.com/dnvriend/akka-persistence-jdbc/releases since this page shows all releases with the latest on top (the draft releases are hidden for the public)